app/__tests__/app.test.js

```diff
@@ -122,6 +81,42 @@ describe("GET /articles/:article_id", () => {
     })
 })
 
+describe("GET /api/articles", () => {
+    describe("-- functionality tests", () => {
+        test("200: responds with 200 status code and all articles, articles should be in descending date order, have a comment count and no body key", () => {
```

Comment:
you're testing quite a few different things here, you should split this test into two - one for the articles having the right keys and one to test the sorting